model/testresult/testresult.go

```diff
@@ -160,3 +160,51 @@ func TestResultsQuery(taskIds []string, testId, testName, status string, limit,
 
 	return q
 }
+
+// TestResultsFilterSortPaginate is a query for returning test results to the taskTests GQL Query.
+func TestResultsFilterSortPaginate(taskIds []string, filter, sortBy string, sortDir, page, limit, execution int) ([]TestResult, error) {
```

Comment:
@john-m-liu I don't think we will have a filter functionality based solely on task status in the most recent designs. The filter params represents whatever the user types into the filter/search bar.

Reply:
Not really. This is part of the DB model, which should be agnostic of what the UI is doing. I'm also not a fan of that design, since the status filter should be a select control whereas the test name filter should be a text search

Reply:
makes sense